.github/renovate.json5

```diff
@@ -497,10 +498,12 @@
       ],
       "postUpgradeTasks": {
         "commands": [
+          "make -C images update-envoy-image",
           "make -C install/kubernetes",
           "make -C Documentation update-helm-values"
         ],
         "fileFilters": [
+          "images/cilium/Dockerfile",
```

Comment:
I think we might need to add some restrictions into here so that we don't update major versions on older stable branches for envoy. Like we do for golang:
```json
    {
      "matchPackageNames": [
        "docker.io/library/golang",
        "go"
      ],
      "allowedVersions": "<1.22",
      "matchBaseBranches": [
        "v1.16",
        "v1.15"
      ]
    },
    {
      "matchPackageNames": [
        "docker.io/library/golang",
        "go"
      ],
      "allowedVersions": "<1.22",
      "matchBaseBranches": [
        "v1.14",
        "v1.13"
      ]
```